.htaccess

```
@@ -1,8 +1,20 @@
 <IfModule mod_headers.c>
     Header set Access-Control-Allow-Origin "*"
 </IfModule>
+<IfModule mod_mime.c>
+    AddType font/ttf .ttf
+    AddType font/woff .woff
+    AddType font/woff2 .woff2
+    AddType font/otf .otf
+    AddType application/octet-stream .eot
+    AddType application/x-apple-aspen-config .mobileconfig
+    AddType image/svg+xml .svg
+    AddType application/javascript .js 
+
+    AddCharset UTF-8 .js
+</IfModule>
 <IfModule mod_rewrite.c>
     RewriteEngine on
 
-    RewriteRule ^(cdnrefresh.php|Dockerfile|web.config|.git|App_Data)$ - [R=404,L]
+    RewriteRule ^(build.sh|cdnrefresh.php|Dockerfile|web.config|.git|.htaccess|App_Data)$ - [R=404,L]
```

```suggestion
    RewriteRule ^(build.sh|Dockerfile|.git|resources/)$ - [R=404,L]
```

* I believe .htaccess will always be [blocked](https://stackoverflow.com/a/36482767/1836776) by Apache default setup.
* cdnrefresh.php is gone
* We should also delete web.config from this branch.
* We should also delete .github from this branch.
* App_Data will also go in its entirety
* resources/ folder should not be accessible